script.js

```diff
@@ -1,11 +1,28 @@
 // Create an array called `colors` that contains three colors.
 
+var colors = ["green", "yellow", "red"];
+
 // Log the last item in the array to the console.
 
+console.log(colors[2]);
+
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
 
+for(i=0; i<colors.length; i++)
+  console.log("My favorite color is " + colors[i]);
+
 // Create an `instructor` object that contains three key-value pairs.
 
+var instructor = {
+  name: "blake",
+  gender: "male",
+  age: 110
+}
+
 // Add a `facial-hair` property to `instructor`.
 
+instructor["facial-hair"] = "none";
+
 // Log the value of the `facial-hair` property to the console.
+
+console.log(instructor.facial-hair);
```

Comment:
have to use `instructor["facial-hair"]`  like you did above (the hyphen will cause an error with `.`: the computer will try to subtract `hair` from `instructor.facial`)